src/components/ActionTooltip.vue

```diff
@@ -164,8 +172,11 @@ export default {
     name () { return this[`Name_${this.lang}`] },
     icon () { return `https://xivapi.com${this.Icon}` },
     actionCategory () { return this.ActionCategory ? this.ActionCategory[`Name_${this.lang}`] : '' },
+    actionNonCombatant () { return [6, 7].includes(this.ActionCategory.ID) },
     description () { return this[`Description_${this.lang}`].replace(/\n\n/g, '<br/>') },
     classJobCategory () { return this.ClassJobCategory ? this.ClassJobCategory[`Name_${this.lang}`] : '' },
+    costType () { return resourceMapping[this.PrimaryCostType] },
```

Comment:
Can we guarantee that this will never be out of bounds? Will this require some guards / defaults?

Reply:
It can be out of bounds but it doesn't matter. The value is only used when costShouldDisplay is true, which doesn't rely on costType itself to get a value.